### PR TITLE
fix: support leftWord, rightWord, deleteWord shortcuts on Windows

### PR DESCRIPTION
**Pull request by YannickPferr**
Release Notes
-------------

New Features
- support `Ctrl + Left`, `Ctrl + Right` and `Ctrl + Backspace` shortcuts on Windows in `confluent flink shell`

Checklist
---------
- [x] Leave this box unchecked if features are not yet available in production

What
----

References
----------

Test & Review
-------------

### CODE REVIEW

File: pkg/flink/internal/controller/input_controller.go [hunk 1]
```diff
@@ -165,6 +166,20 @@ func (c *InputController) Prompt() (prompt.IPrompt, error) {
 			ASCIICode: []byte{0x1b, 0x7F},
 			Fn:        prompt.DeleteWord,
 		}),
+		// Windows shortcuts
+		prompt.OptionAddKeyBind(prompt.KeyBind{
+			Key: prompt.ControlLeft,
```
Comment (brianstrauch) [inline pkg/flink/internal/controller/input_controller.go:171]: I can also use Ctrl+Left on my Mac, so should we gate this code with `if runtime.GOOS == "windows"`?

Comment (YannickPferr) [reply pkg/flink/internal/controller/input_controller.go]: Yeah makes sense, updated it

File: pkg/flink/internal/controller/input_controller.go [hunk 2]
```diff
@@ -213,3 +178,77 @@ func (c *InputController) toggleSmartCompletion() {
...
+			prompt.OptionAddKeyBind(prompt.KeyBind{
+				Key: prompt.ControlR,
+				Fn: func(b *prompt.Buffer) {
+					c.reverseISearchEnabled = true
+				},
+			}),
+		}, osSpecificBindings...)
+}
+
+func getMacBindings() []prompt.Option {
```
Comment (Guttz) [inline pkg/flink/internal/controller/input_controller.go:216]: You probably want to name this UnixBindings?

File: pkg/flink/internal/controller/input_controller.go [hunk 3]
```diff
@@ -213,3 +178,77 @@ func (c *InputController) toggleSmartCompletion() {
...
+			},
+		),
+	}
+}
+
+func getWindowsBindings() []prompt.Option {
+	return []prompt.Option{
+		prompt.OptionAddKeyBind(
+			prompt.KeyBind{
+				Key: prompt.ControlLeft,
```
Comment (Guttz) [inline pkg/flink/internal/controller/input_controller.go:239]: Is this the default shortcut on windows for navigating words? I thought alt + arrow was broken and that was what we wanted to fix

Comment (Guttz) [reply pkg/flink/internal/controller/input_controller.go:239]: Nop, ctrl + key is the windows default shortcut. Just tested it too

**Review by Guttz — APPROVED** [review]

**Pull request merged**